nemo/utils/export_utils.py

```diff
@@ -213,7 +213,7 @@ def run_ort_and_compare(sess, ort_input, output_example, check_tolerance=0.01):
     from apex.transformer.tensor_parallel.layers import RowParallelLinear, ColumnParallelLinear
     from apex.transformer.functional.fused_softmax import FusedScaleMaskSoftmax
 
-    def replace_FusedLayerNorm(n: nn.Module) -> Optional[nn.BatchNorm2d]:
+    def replace_FusedLayerNorm(n: nn.Module) -> Optional[nn.LayerNorm]:
```

Comment:
Revert for now, it's mixed case use type is incorrect anyway

Reply:
Reverted